ddtrace/profiling/collector/stack.pyx

```diff
@@ -29,23 +35,9 @@ from ddtrace.utils import formats
 FEATURES = {
     "cpu-time": False,
     "stack-exceptions": False,
-    "gevent-tasks": False,
```

Comment:
This seems like a breaking change without a release note (unless we're really sure that no one is using this configuration)

```python
if FEATURES["gevent-tasks"]:
    ...
```

or similar in their code